src/main/java/games/strategy/engine/data/properties/FileProperty.java

```diff
@@ -167,7 +166,7 @@ public String getDescription() {
         return Arrays.toString(acceptableSuffixes);
       }
     });
-    final int rVal = fileChooser.showOpenDialog(MainFrame.getInstance());
+    final int rVal = fileChooser.showOpenDialog(null);
```

Comment:
Much better than my suggestion to simply expose `GameRunner#mainFrame`.  I like the direction this is going.  Thanks for the extra effort to do this, @DanVanAtta!